packages/orchestrator-ui-components/src/components/WfoSubscription/WfoRelatedSubscriptions.tsx

```diff
@@ -70,82 +73,78 @@ export const WfoRelatedSubscriptions = ({
 
-    const tableColumns: WfoTableColumns<RelatedSubscription> = {
+    const tableColumns: WfoTableColumnConfig<RelatedSubscription> = {
         subscriptionId: {
-            field: 'subscriptionId',
-            name: t('id'),
+            columnType: ColumnType.DATA,
+            label: t('id'),
             width: '100',
-            render: (value) => <WfoFirstPartUUID UUID={value} />,
-            filterable: false,
+            renderData: (value) => <WfoFirstPartUUID UUID={value} />,
         },
         description: {
-            field: 'description',
-            name: t('description'),
-            render: (value, record) => (
+            columnType: ColumnType.DATA,
+            label: t('description'),
+            renderData: (value, record) => (
                 <Link
                     target="_blank"
                     href={`${PATH_SUBSCRIPTIONS}/${record.subscriptionId}`}
                 >
                     {value}
                 </Link>
             ),
-            filterable: false,
         },
         status: {
-            field: 'status',
-            name: t('status'),
+            columnType: ColumnType.DATA,
+            label: t('status'),
             width: '130',
-            render: (value) => <WfoSubscriptionStatusBadge status={value} />,
-            filterable: false,
+            renderData: (value) => (
+                <WfoSubscriptionStatusBadge status={value} />
+            ),
         },
         insync: {
-            field: 'insync',
-            name: t('insync'),
+            columnType: ColumnType.DATA,
+            label: t('insync'),
             width: '60',
-            render: (value) => <WfoInsyncIcon inSync={value} />,
-            filterable: false,
+            renderData: (value) => <WfoInsyncIcon inSync={value} />,
         },
         customer: {
-            field: 'customer',
-            name: t('customer'),
-            render: (customer) => customer.fullname,
-            sortable: false,
-            filterable: false,
+            columnType: ColumnType.DATA,
+            label: t('customer'),
+            renderData: (customer) => customer.fullname,
         },
         product: {
-            field: 'product',
-            name: t('tag'),
+            columnType: ColumnType.DATA,
+            label: t('tag'),
             width: '150',
-            render: (product) => product.tag,
-            filterable: false,
+            renderData: (product) => product.tag,
         },
         startDate: {
-            field: 'startDate',
-            name: t('startDate'),
+            columnType: ColumnType.DATA,
+            label: t('startDate'),
             width: '100',
-            render: (value) => parseDateToLocaleDateString(parseDate(value)),
-            filterable: false,
+            renderData: (value) =>
+                parseDateToLocaleDateString(parseDate(value)),
         },
     };
 
+    const onUpdatePageIndex =
```

Comment:
Small nitpick but I would rename this to onChangeItemsPerPage right away to you pass likt that in line 140 right away

Reply:
done